### DISCUSSION TITLE
Data Points are 60 seconds apart - Started Out 1 second intervals.

### DISCUSSION THREAD

**Discussion by poorredneck** [jertel/vuegraf#101] [Q&A] [answered]
Hello,

First off thank you for all your work on Vuegraf. The Emporia app leaves alot to be desired, and I've used Influx/Grafana in the past with a RTL-SDR and my old meter. New meter is not compatible now (encrypted). Emporia and Vuegraf to the rescue!

Just got everything running. Windows 10, using the Docker containers. Using the Influx 1.8 in the revised Readme file (btw, might want to update the readme on the docker site). For the first few minutes I was getting data updates every second as expected. As I'm setting up my dashboard, now each graph is only updating every 60 seconds. I'm sure it's something stupid I'm overlooking but I've pulled my hair out trying to figure it out. 

Thank you for any assistance!
Jon

**Comment by jertel** [comment, marked as answer]
Hello, 

Please see #57 (and the linked discussion within for even more details) to get a better understanding of the Vuegraf update timing. If you are seeing new data show up every minute, then it's working correctly. However, if you are not seeing the per second data show up within an hour, then there may be something misconfigured. 

If you're using the example configuration file as your template then there should not be an issue, as that example doesn't override the default timing. I advise against overriding the defaults as it can swamp the Emporia server with excessive requests.

**Reply by poorredneck** [reply]
You are absolutely correct, I mis-read that post. Now that it has been up and running for 12 hours or so I can see that the older data is populated on a per second basis, and only the most recent is per minute. Sincere apologies, I jumped the gun and panicked.
